spec/controllers/idv/by_mail/request_letter_controller_spec.rb

```diff
@@ -222,23 +223,26 @@
 
         expect(@analytics).to have_logged_event(
           'IdV: USPS address letter requested',
-          resend: true,
-          phone_step_attempts: 1,
-          first_letter_requested_at: pending_profile.gpo_verification_pending_at,
-          hours_since_first_letter: 24,
-          proofing_components: nil,
-          **ab_test_args,
+          hash_including(
+            resend: true,
+            phone_step_attempts: 1,
+            first_letter_requested_at: pending_profile.gpo_verification_pending_at,
+            hours_since_first_letter: 24,
+            **ab_test_args,
+          ),
         )
```

Comment:
Rather than adding 

```ruby
active_profile_idv_level: nil,
pending_profile_idv_level: nil,
proofing_components: nil,
```

here, I switched it (and several other specs) to use `hash_including`. My feeling is that these tests aren't really testing these extra analytics arguments--those should be covered by a combination of the `Idv::AnalyticsEventsEnhancer` spec and the analytics feature spec.